js/src/Builder.js

```diff
@@ -873,6 +1039,10 @@ function _set_up_menu (menu_selection, map, key_manager, keys, enable_editing,
                 id: 'text-mode-menu-button',
                 text: 'Text mode',
                 key_text: (enable_keys ? ' (T)' : null) })
+      .button({ key: keys.difference_mode, // TODO: this button is not working
```

Comment:
it's less complicated to add a button that's not considered a 'mode'. Look at the search/find button and bar as an example.